mne/io/hitachi/tests/test_hitachi.py

```diff
@@ -195,21 +196,46 @@ def test_hitachi_basic(preload, version, n_ch, n_times, lowpass, sex, date,
         contents = contents.replace(b'\n', end)
     with open(fname, 'wb') as fid:
         fid.write(CONTENTS[version])
-    raw = read_raw_hitachi(fname, preload=preload)
+    raw = read_raw_hitachi(fname, preload=preload, verbose=True)
     data = raw.get_data()
     assert data.shape == (n_ch, n_times)
     assert raw.info['sfreq'] == 10
     assert raw.info['lowpass'] == lowpass
     assert raw.info['subject_info']['sex'] == sex
     assert np.isfinite(raw.get_data()).all()
     assert raw.info['meas_date'] == dt.datetime(*date, tzinfo=dt.timezone.utc)
-    # distances
+    # bad distances
     distances = source_detector_distances(raw.info)
-    want = [0.] * ((n_ch - 4) // 2)
-    assert_allclose(distances[::2], want, atol=0.)
+    want = [0.] * (n_ch - 4)
+    assert_allclose(distances, want, atol=0.)
+    raw_od_bad = optical_density(raw)
+    with pytest.warns(RuntimeWarning, match='will be zero'):
+        beer_lambert_law(raw_od_bad)
+    # good distances
+    mon = make_standard_montage('standard_1020')
+    if version == '1.18':
+        need = sum(([f'S{ii}', f'D{ii}'] for ii in range(1, 9)), [])[:-1]
+        have = 'F3 FC3 C3 CP3 P3 F5 FC5 C5 CP5 P5 F7 FT7 T7 TP7 P7'.split()
+        dist_tol = 0.01
+        bl_min = 1e-6
+    else:
+        need = sum(([f'S{ii}', f'D{ii}'] for ii in range(1, 18)), [])[:-1]
+        have = mon.ch_names[:len(need)]  # wrong but good enough for tests
```

Comment:
@rob-luke in particular if you have the correct mapping for the data file you supplied that would be great, as we could fix the `dist_tol` and `bl_min` tolerances

Reply:
Sorry I don't have any more information about the dataset as I didnt measure it, it was provided to me.

Reply:
But this means you have no information about positions, so you aren't using BLL? If so that's fine and this test can just stay as is, but it's also a bit :(

I can probably figure out some 1020 positions from the standard Hitachi configuration so that the distances are at least sane, though. So if you can't make it easy on me by providing them, then I'll at least sort out a reasonable possible arrangement :)

Reply:
I never processed the data. I could ask the owner for info if required. I know it was a standard montage array.

Reply:
Give me a few days and ill try and track down more details

Reply:
I documented the layout structure and am pushing a commit with a reasonable choice